service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/PatientMapper.java

```diff
@@ -1,19 +1,166 @@
 package uk.nhs.adaptors.oneoneone.cda.report.mapper;
 
-import static org.hl7.fhir.dstu3.model.IdType.newRandomUuid;
+import lombok.AllArgsConstructor;
 
+import org.hl7.fhir.dstu3.model.Address;
+import org.hl7.fhir.dstu3.model.CodeableConcept;
+import org.hl7.fhir.dstu3.model.ContactPoint;
+import org.hl7.fhir.dstu3.model.Enumerations;
+import org.hl7.fhir.dstu3.model.Extension;
+import org.hl7.fhir.dstu3.model.HumanName;
 import org.hl7.fhir.dstu3.model.Patient;
+import org.hl7.fhir.dstu3.model.Reference;
+import org.hl7.fhir.dstu3.model.StringType;
+import org.hl7.fhir.dstu3.model.Organization;
+
 import org.springframework.stereotype.Component;
 
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01LanguageCommunication;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01Patient;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01PatientRole;
+import uk.nhs.adaptors.oneoneone.cda.report.util.NodeUtil;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.hl7.fhir.dstu3.model.IdType.newRandomUuid;
+
 @Component
+@AllArgsConstructor
 public class PatientMapper {
-    //TODO
-    // This is just a placeholder for the mapper. EpisodeOfCare requires a reference to a patient for message to be valid
```

Comment:
This //TODO could be removed

Reply:
This error was caused by merge conflict.